stories/core/Table.stories.tsx

```diff
@@ -49,6 +49,7 @@ export default {
   argTypes: {
     columns: { control: { disable: true } },
     isSelectable: { control: { disable: true } },
+    hasColumnManager: { control: { disable: true } },
```

Comment:
I disabled this control because I heard it may be impossible to switch the table columns dynamically (the same reason why isSelectable is disabled)

Reply:
Can someone explain more apart its `react-table` thing? :D

Reply:
@bentleyvk @mayank99

Reply:
I think because it is made through `react-table` hooks and therefore can't be undone.

Reply:
column manager is no longer a table prop, so this has been removed